### DISCUSSION TITLE
rebuildAll.js not respecting z-index

### DISCUSSION THREAD

**Discussion by BakeSloth** [nftchef/art-engine#237] [Help + Q&A]
_dna.json has all the zindex info baked in
example: 
"0.0.0:**z1**,Blue#6.png*1.0.4:**z2**,Pink#90.png*5.0.6:**z6**,Bruised#20.png*9.0.0:**z10**,Rattle#5.png",

but for some reason the rebuildAll.js script ignores it. 
it looks like its pulling layer order directly from config.js layersOrder array without sorting by zindex.

does anyone have any idea on how to fix this? 
I need to fix the collection but cant with this issue, please help.

**Comment by ranovr** [comment]
Bumping. I have this too.

**Comment by nftchef** [comment]
This is been fixed 🙇
